Fixes on French translation file

CyrilleB79 opened the pull request:
Ciao a tutti!

Here are small fixes on the French readme file.
* Fixed formatting issues with `+` character left over
* Replaced "shift" by the French "maj".
* Harmonization of non-breaking spaces usage
* List formatting fix (add blank line before list): to be ported to English file in a separate work (I do not want to mix English doc edition with translated ones).

Please note that the English file is not up-to-date with respect to French or Italian file, mores specifically regarding the paragraph describing the "Nao" menu; this is not fixed in the current PR.

Commit:
Fixes on French translation file

B-Benanou commented:
Hi!
Not only the English file needs updating, but in the French Readme file line feeds are replaced by spaces and undue (+) plus signs, breaking the markdown structure and causing incorrect HTML formatting.
Non-breaking spaces are not systematically present where needed; these need to be either removed altogether or put where they should be.
I see that  a (²) superscript2 sign is present after the name of the program Explorer² whereas it doesn't exist in the English file that's actually shipped with the NVDA NAO. I mean NVDA Advanced OCR V2025.1.0.
Would be nice if documentation and French translation where to be reviwed.
Thanks for reading.

CyrilleB79 committed:
Non-breaking space harmonization

CyrilleB79 committed:
Fix list formatting

CyrilleB79 commented:
I have added more fixes and updated the initial description.

@B-Benanou please let me know if things in the French file are remaining.

This PR does not aim to fix issues in the English file. More specifically, the "²" character has been copied since used in the English file.

For issues in the English file, see #53.

B-Benanou commented on addon/doc/fr/readme.md:
```diff
@@ -1,45 +1,46 @@
 L'extension fonctionne sur les systèmes Windows 10 et Windows 11, car ils ont des capacités OCR intégrées. 
 Nao est compatible à partir de la version 2019.3 de NVDA, donc n'utilisez pas une version plus ancienne du lecteur d'écran.
-Veuillez noter que Nao fonctionne avec l'explorateur Windows, sur le bureau ou avec les gestionnaires de fichiers Total Commander ou xplorer² ; n'utilisez pas d'autres logiciels comme 7zip ou Winrar , car ils ne sont pas pris en charge.
+Veuillez noter que Nao fonctionne avec l'explorateur Windows, sur le bureau ou avec les gestionnaires de fichiers Total Commander ou xplorer² ; n'utilisez pas d'autres logiciels comme 7zip ou Winrar , car ils ne sont pas pris en charge.
```
Undue space before comma.

CyrilleB79 replied:
OK:

```suggestion
Veuillez noter que Nao fonctionne avec l'explorateur Windows, sur le bureau ou avec les gestionnaires de fichiers Total Commander ou xplorer² ; n'utilisez pas d'autres logiciels comme 7zip ou Winrar, car ils ne sont pas pris en charge.
```

B-Benanou commented on addon/doc/fr/readme.md:
Bonjour @CyrilleB79, Buon giorno tutti,
The French translation looks a lot nicer now.
Mind that space before comma where WinRar is stated.
Thanks for fixing addon/doc/fr/readme.md.

CyrilleB79 committed:
Update addon/doc/fr/readme.md
